### PR TITLE
Add nice issue templates

### PR DESCRIPTION
**Pull request by MichaelKohler**

### CODE REVIEW

**Comment by MichaelKohler** [comment]
Once we support scheduled tweets, we should create a template for it as well.

File: .github/ISSUE_TEMPLATE/retweet.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this suggestion! Once submitted we will review it and publish it if everything is okay.
+  - type: input
+    id: description
+    attributes:
+      label: Content Description
+      description: What is this tweet about? Please describe the purpose in detail. This part will not be part of the tweet!
```
Comment (freaktechnik) [inline .github/ISSUE_TEMPLATE/retweet.yml]: > This will not be part of the tweet

Should be enough.

File: .github/ISSUE_TEMPLATE/retweet.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: Retweet
+description: Suggest a retweet of another tweet
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this suggestion! Once submitted we will review it and publish it if everything is okay.
+  - type: input
```
Comment (freaktechnik) [inline .github/ISSUE_TEMPLATE/retweet.yml]: I'm wondering if this should be a textarea instead? I guess both are fine.

**Review by freaktechnik — APPROVED** [review]

**Comment by freaktechnik** [comment]
> Once we support scheduled tweets, we should create a template for it as well.

I'd just add an optional field to the normal templates (with a placeholder of the format, of course)

File: .github/ISSUE_TEMPLATE/retweet.yml (hunk 1 above)
Comment (MichaelKohler) [reply .github/ISSUE_TEMPLATE/retweet.yml]: Hah, absolutely!

File: .github/ISSUE_TEMPLATE/retweet.yml (hunk 2 above)
Comment (MichaelKohler) [reply .github/ISSUE_TEMPLATE/retweet.yml]: Yeah, if we ask for details, we should also make it easy... Good point!

**Pull request merged**